requirements.txt

```diff
@@ -10,4 +10,5 @@ ascat>=1.0
 ismn>=0.3
 configparser
 pykdtree
-repurpose>=0.7.1
\ No newline at end of file
+repurpose>=0.7.1
```

Comment:
Do we keep the requirements.txt file with the new structure`?

Reply:
this will be gone once pyscaffold merge request is done